Makefile

```diff
@@ -163,10 +163,15 @@ build-dirs:
 	@mkdir -p bin/$(ARCH)
 	@mkdir -p .go/src/$(PKG) .go/pkg .go/bin .go/std/$(ARCH)
 
-clean: container-clean bin-clean
+clean: container-clean bin-clean  ## cleans the repository's temporary files
 
 container-clean:
 	rm -rf .container-* .dockerfile-* .push-*
 
 bin-clean:
 	rm -rf .go bin
+
+help:
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
```

Comment:
can you linewrap this?  Something like:

```
grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) \
    | sort \
    | awk ' \
        BEGIN {FS = ":.*?## "}; \
        {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}; \
    '
```

Reply:
Maybe it would be cleaner to do something like:

```
# @HELP clean: cleans the repository's temporary files
clean: container-clean bin-clean

# @HELP test: runs tests for the project
test: build-dirs
...
```

...and then grep and process those lines?  sorting them makes it harder, but I can live with one-liners